hack/testing/check-EFK-running.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-set -o errexit
-set -o nounset
-set -o pipefail
```

Comment:
These settings are extremely useful - why get rid of them?  Does init.sh set these flags?